easybuild/easyblocks/i/imkl.py

```diff
@@ -77,6 +77,34 @@ def prepare_step(self, *args, **kwargs):
             super(EB_imkl, self).prepare_step(*args, **kwargs)
         else:
             super(EB_imkl, self).prepare_step(*args, **kwargs)
+        if self.cfg['interfaces']:
+            cdftlibs = ['fftw2x_cdft']
+            if LooseVersion(self.version) >= LooseVersion('10.3'):
+                cdftlibs.append('fftw3x_cdft')
+            mpi_spec = None
+            # check whether MPI_FAMILY constant is defined, so mpi_family() can be used
+            if hasattr(self.toolchain, 'MPI_FAMILY') and self.toolchain.MPI_FAMILY is not None:
+                mpi_spec_by_fam = {
+                    toolchain.MPICH: 'mpich2',  # MPICH is MPICH v3.x, which is MPICH2 compatible
+                    toolchain.MPICH2: 'mpich2',
+                    toolchain.MVAPICH2: 'mpich2',
+                    toolchain.OPENMPI: 'openmpi',
+                }
+                mpi_fam = self.toolchain.mpi_family()
+                mpi_spec = mpi_spec_by_fam.get(mpi_fam)
+                self.log.debug("Determined MPI specification based on MPI toolchain component: %s" % mpi_spec)
```

Comment:
@bartoldeman While touching this, please change this to:

```python
self.log.debug("...: %s", mpi_spec)
```

(`, ` rather than ` % `; same below)